core/env/clear_spec.rb

```diff
@@ -3,8 +3,9 @@
 describe "ENV.clear" do
   it "deletes all environment variables" do
     orig = ENV.to_hash
+    env_object_id = ENV.object_id
     begin
-      ENV.clear
+      ENV.clear.object_id.should == env_object_id
```

Comment:
There is a matcher to do this more clearly (see https://github.com/ruby/spec/blob/master/CONTRIBUTING.md#matchers-and-expectations): 
```ruby
ENV.clear.should equal(ENV)
```
Could you use it?

Reply:
Thanks, eregon.  I will update to use the matcher.  Also, I will study the matchers at the link.